File: Python/Logic-2/lone_sum.py
def lone_sum(a, b, c):
  sum = a + b + c
  
  if (a == c and b == c and a == b):
    return 0
    
  if (a == c):
    sum -= a + c
  if (b == c):
    sum -= b + c
  if (a == b):
    sum -= b + a
    
  return sum

File: Java/Logic-2/loneSum.java
public int loneSum(int a, int b, int c) {
  if (a == b && b == c && a == c) return 0;
  if (a == b && b != c && a != c) return c;
  if (a != b && b != c && a == c) return b;
  if (a != b && b == c && a != c) return a;
  return a + b + c;
}
